ui.go

```diff
@@ -1445,6 +1447,9 @@ func (ui *ui) readNormalEvent(ev tcell.Event, nav *nav) expr {
 		log.Printf("Got EventError: '%s' at %s", tev.Error(), tev.When())
 	case *tcell.EventInterrupt:
 		log.Printf("Got EventInterrupt: at %s", tev.When())
+	case *tcell.EventFocus:
+		ui.focused = tev.Focused
+		return &callExpr{"redraw", nil, 1}
```

Comment:
I don't think calling `redraw` is appropriate here. I tested with image previews just now and this actually invokes the `cleaner` program, causing the image to flicker when the focus changes (can be quite annoying). Perhaps it would be better to just use `return draw` here.

Reply:
Yes, I tested it with `return draw` and this works as well.